server/src/test/java/org/jboss/as/controller/persistence/RemoteGitPersistenceResourceTestCase.java

```diff
@@ -59,12 +62,16 @@ public void createDirectoriesAndFiles() throws Exception {
     public void deleteDirectoriesAndFiles() throws Exception {
         if (remoteRepository != null) {
             remoteRepository.close();
+            remoteRepository = null;
         }
         if (repository != null) {
             repository.close();
+            repository = null;
         }
-        delete(remoteRoot.getParent().toFile());
-        delete(root.getParent().toFile());
+        RepositoryCache.clear();
+        TimeUnit.MILLISECONDS.sleep(500);
+        FileUtils.delete(remoteRoot.getParent().toFile(), FileUtils.RECURSIVE | FileUtils.RETRY | FileUtils.SKIP_MISSING);
+        FileUtils.delete(root.getParent().toFile(), FileUtils.RECURSIVE | FileUtils.RETRY | FileUtils.SKIP_MISSING);
```

Comment:
@ehsavoie FYI, this one is still pending to modify with the latest changes ignoring the error on Windows